testing/go-workspace/tests/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+load(":go-test.bzl", "go_test_suite")
+
+package(default_testonly = 1)
+
+# NOTE Keep in sync with testing/go-bzlmod/tests/BUILD.bazel
```

Comment:
Would it be reasonable to make one of these files a symlink to the other so they don't have to be kept in sync?

Reply:
Ah I see now, that makes sense. I was comparing the two files with `git diff` instead of plain `diff` :facepalm:
```
ben@thinkpad (bzlmod-go) 140257f ~/git/rules_nixpkgs $ git diff testing/go-workspace/tests/BUILD.bazel testing/go-bzlmod/tests/BUILD.bazel
ben@thinkpad (bzlmod-go) 140257f ~/git/rules_nixpkgs $ diff testing/go-workspace/tests/BUILD.bazel testing/go-bzlmod/tests/BUILD.bazel
1c1
< load("@io_bazel_rules_go//go:def.bzl", "go_binary")
---
> load("@rules_go//go:def.bzl", "go_binary")
6c6
< # NOTE Keep in sync with testing/go-bzlmod/tests/BUILD.bazel
---
> # NOTE Keep in sync with testing/go-workspace/tests/BUILD.bazel
```